src/pages/FlowRun/Actions.vue

```diff
@@ -59,11 +59,18 @@ export default {
       if (oldVal.state === 'Scheduled' && newVal.state !== 'Scheduled') {
         this.isRunningNow = false
       }
+      if (FINISHED_STATES.includes(newVal.state)) {
```

Comment:
I think to avoid excessive queries for finished runs we should check that `newVal` and `oldVal` are different:
 ```suggestion
      if (newVal.state !== oldVal.state && FINISHED_STATES.includes(newVal.state)) {
```

Reply:
Good point - I've added.

Reply:
Sorry, it looks like my suggestion caused a linting issue that's preventing the build 🤦